bundle-workflow/src/test_workflow/put_file_into_S3.py

```diff
@@ -0,0 +1,361 @@
+# !/usr/bin/env python3
```

Comment:
This file has too many jobs, its parsing input, transformation json into html, and writing out to a file.  These should all be seperate tasks, and I would advocate for the file uploading to be 1 pull request, and the transformation to be in another.